dist/octogerrit.css

```diff
@@ -668,8 +668,11 @@ div.com-google-gerrit-client-change-ChangeScreen_BinderImpl_GenCss_style-idAndSt
 table.com-google-gerrit-client-change-FileTable-FileTableCss-table {
   width: 100%;
 }
+button.com-google-gerrit-client-change-ChangeScreen_BinderImpl_GenCss_style-highlight {
+  background-image: -webkit-linear-gradient(top, #4d90fe, #4d90fe);
```

Comment:
check out github's 'Start a Review' and 'Review Changes' buttons. they've got a nice, subtle gradient shift from top to bottom with a border color change to match. I'd like to see this change follow suit so we can keep the elements of the ux in the same spirit and feel.